src/components/Discover/TrendingTokens.tsx

```diff
@@ -381,21 +404,45 @@ function TrendingTokenRow({ token }: { token: TrendingToken }) {
                   alignItems: IS_IOS ? 'baseline' : 'flex-end',
                   flexDirection: 'row',
                   gap: 6,
-                  height: 10,
-                  maxWidth:
-                    DEVICE_WIDTH -
-                    40 - // horizontal list padding
-                    (40 + 12) - // token icon width + 12px gap
-                    70, // approx. % price change width
+                  height: 12,
+                  width: TOKEN_DETAILS_WIDTH,
+                  overflow: 'hidden',
```

Comment:
```suggestion
```
This causes clipping on the token prices with subscripts and some characters ("y") on the token name